config/config.go

```diff
@@ -7,29 +7,46 @@ import (
 )
 
 // Config maps org or repo to LinterConfig
+//type Config map[string]map[string]Linter
+
 type Config map[string]map[string]Linter
 
 type Linter struct {
 	// Enable is whether to enable this linter, if false, linter still run but not report.
-	Enable  bool     `json:"enable,omitempty"`
-	WorkDir string   `json:"workDir,omitempty"`
-	Command string   `json:"command,omitempty"`
-	Args    []string `json:"args,omitempty"`
+	Enable  *bool    `json:"enable,omitempty" yaml:"enable,omitempty"`
+	WorkDir string   `json:"workDir,omitempty" yaml:"workDir,omitempty"`
+	Command string   `json:"command,omitempty" yaml:"command,omitempty"`
+	Args    []string `json:"args,omitempty" yaml:"args,omitempty"`
 }
 
 // NewConfig returns a new Config.
 func NewConfig(conf string) (Config, error) {
-	var c Config
 	f, err := os.ReadFile(conf)
 	if err != nil {
 		return nil, err
 	}
 
+	c := Config{}
 	if err = yaml.Unmarshal(f, &c); err != nil {
 		return nil, err
 	}
 
-	// TODO: 应该默认开启staticcheck？
+	defaultEnable := true
+	for _, v := range c {
+		for _, val := range v {
+			if val.Enable == nil {
+				val.Enable = &defaultEnable
+			}
+		}
+	}
+
+	if len(c) == 0 {
```

Comment:
currently， `config` only work as a `filter`, so the only scenario that currently requires configuration is to some customize